samples/Samples.Server/Startup.cs

```diff
@@ -65,7 +65,7 @@ public void Configure(IApplicationBuilder app)
         app.UseWebSockets();
 
         app.UseGraphQLWebSockets<ChatSchema>();
-        app.UseGraphQL<ChatSchema, GraphQLHttpMiddlewareWithLogs<ChatSchema>>();
+        app.UseGraphQL<GraphQLHttpMiddlewareWithLogs<ChatSchema>>("/graphql", new GraphQLHttpMiddlewareOptions());
```

Comment:
```suggestion
        app.UseGraphQL<GraphQLHttpMiddlewareWithLogs<ChatSchema>>();
```

Reply:
This is because the extension method isn't tied to any specific options class; you can pass in any options class.  Since it doesn't know what options class you're using, it won't try to inject one by default.  The old code did not allow any options classes to be passed in here, but also required the options to be registered in DI.  If you do register the options with DI, then you don't need to pass them in here.